8Knot/cache_manager/db_init.py

```diff
@@ -228,9 +228,10 @@ def _create_application_tables() -> None:
         cur.execute(
             """
             CREATE UNLOGGED TABLE IF NOT EXISTS cntrb_per_file_query(
-                file_path text,
```

Comment:
since the order changes, do we need to make any changes to the other heatmap visualizations?

Reply:
From what I was seeing the other visualizations were actually breaking before this change

Reply:
++